### PR TITLE
Add support for bokeh static command

### PR DESCRIPTION
**Pull request by mattpap**
Run `bin/bokeh static` and you will get bokehjs served under `http://localhost:5006/static/`. Address (host), port and log level are configurable. This uses the "full" bokeh server, but configures only `/static/` url, so no apps, websockets, etc.

### CODE REVIEW

**Comment by bryevdv** [comment]
Was failing for a dumb reason. Should pass now then we can merge.

File: bokeh/command/subcommands/static.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+import logging
+log = logging.getLogger(__name__)
+
+from bokeh.server.server import Server
+from bokeh.util.string import nice_join
+
+from ..subcommand import Subcommand
+
+LOGLEVELS = ('debug', 'info', 'warning', 'error', 'critical')
```
Comment (bryevdv) [inline bokeh/command/subcommands/static.py]: This should be factored out somewhere.

File: bokeh/command/subcommands/static.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+class Static(Subcommand):
+    ''' Subcommand to launch the Bokeh static server. '''
+
+    name = "static"
+
+    help = "Run a Bokeh static server (hosting bokehjs' static files)"
```
Comment (bryevdv) [inline bokeh/command/subcommands/static.py]: suggest `help= "Serve BokehJS static JavaSript and CSS assets"`

File: bokeh/command/subcommands/static.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        )),
+
+        ('--log-level', dict(
+            metavar='LOG-LEVEL',
+            action  = 'store',
+            default = 'info',
+            choices = LOGLEVELS,
+            help    = "One of: %s" % nice_join(LOGLEVELS),
+        )),
+    )
```
Comment (bryevdv) [inline bokeh/command/subcommands/static.py]: This should be codified in a test (inching towards locking down API and CLI specifications)

**Comment by bryevdv** [comment]
@mattpap a few small comments

**Comment by mattpap** [comment]
@bryevdv, I think your comments were addressed. I resolved conflicts. This is ready.

**Comment by bryevdv** [comment]
test failures seem to be real, not sure whats going on

**Comment by bryevdv** [comment]
LGTM, merging

**Pull request merged**